wp-instagram-widget.php

```diff
@@ -105,7 +106,7 @@ function widget( $args, $instance ) {
 					if ( locate_template( $template_part ) !== '' ) {
```

Comment:
For this level of customisation you could use your own template part as checked for here ^

By default it looks for a file called `parts/wp-instagram-widget.php` within your theme and that file can contain a complely custom list item:

```
<?php
echo '<li class="' . esc_attr( $liclass ) . '"><a href="' . esc_url( $item['link'] ) . '" target="' . esc_attr( $target ) . '"  class="' . esc_attr( $aclass ) . '"><img src="' . esc_url( $item[$size] ) . '"  alt="' . esc_attr( $item['description'] ) . '" title="' . esc_attr( $item['description'] ) . '"  class="' . esc_attr( $imgclass ) . '"/></a></li>';
```

Reply:
This would be inside the theme directory. The folder and filename are both filterable too.

Just another option if you want more control

Reply:
Is there a concern about adding more filters to the plugin? 

We've been recommending WPIW for years, and child themes aren't updated nearly as easily or frequently as plugins. If we added a parts folder to customize this, it would take months (maybe years) to roll out, same with any modification we had to make.

Reply:
Not a huge concern, especially as the plugin has just been yanked from the .org plugin repository. I just wanted to check that the other option was explored.

Reply:
Giving this some more thought, I am wondering if it will be easier to use an array here than a string.